ecoscope/io/eetools.py

```diff
@@ -95,12 +97,7 @@ def label_gdf_with_img(gdf=None, img=None, region_reducer=None, scale=500.0):
 
 
 def _match_gdf_to_img_coll_ids(
```

Comment:
I prefer having the function params on separate lines which makes tracking changes to function signatures easier in github, and also prevents exceeding pep8 requirements for longer functions.

Reply:
Looking through the Black documentation, it says you can use this to have it ignore formatting:

# fmt: off

block of code

# fmt: on

Thing is, you have to do this for each section you want it to ignore. At least as of what I have found from my brief search. I am looking to see if there is a way we can universally have it ignore function definitions so we don't have to individually ignore them in the code.

Reply:
Ok interesting. Well, it's not so important that I think we should turn off that formatting rule. Let's stick with it for now. Thanks for looking into it.